flink/scripts/common.sh

```diff
@@ -37,24 +37,36 @@ clean_protocol() {
 
 log_debug() {
   if [ "$cobli_verbosity_level" -ge 3 ]; then
-    echo -e "${cyan_color}${cobli_script}: <DEBUG> $*${no_color}" 1>&2
+    timestamp="$(date --rfc-3339=ns)"
+    prefix="${cyan_color}${timestamp} ${cobli_script}: <DEBUG>"
+    sufix="${no_color}"
+    echo -e "${prefix} $*${sufix}" 1>&2
```

Comment:
I think there could be a function to return the prefix with the green timestamp.
So you can call it like: `echo -e "${prefix("DEBUG")} $*${sufix}" 1>&2`